### PR TITLE
Add new homebrew file via upload

### PR DESCRIPTION
**Pull request by badooga**
Adds new homebrew: Better Greatwyrms by badooga.

### CODE REVIEW

File: creature/badooga; Better Greatwyrms.json [hunk 1]
```diff
@@ -0,0 +1,2138 @@
+{
+	"siteVersion": "1.147.13",
```
Comment (Git-GoR) [inline creature/badooga; Better Greatwyrms.json:2]: Remove this `siteVersion` key to pass the tests.

It's automatically added if you download content from the homebrew builder; used to warn the user about using older files in newer versions of the site. It isn't needed for content in the repository.

**Review by Git-GoR — CHANGES_REQUESTED** [review]
🤏 Small change requested

File: creature/badooga; Better Greatwyrms.json [hunk 2]
```diff
@@ -0,0 +1,2137 @@
...
+				"RCH",
+				"AOE"
+			],
+			"conditionInflict": [
+				"frightened",
+				"grappled",
+				"prone",
+				"restrained"
+			],
+			"conditionInflicted": [
```
Comment (TheGiddyLimit) [inline creature/badooga; Better Greatwyrms.json:234]: Should be `conditionInflict`
Issue repeated a few times throughout the file

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]

File: creature/badooga; Better Greatwyrms.json (hunk 2 above)
Comment (badooga) [reply creature/badooga; Better Greatwyrms.json:234]: You mean, I should remove those entries from all the stat blocks? I believe they came from the Homebrew Builder

Comment (revilowaldow) [reply creature/badooga; Better Greatwyrms.json:234]: So that's where all those have been coming from

Comment (revilowaldow) [reply creature/badooga; Better Greatwyrms.json:234]: Watch out, normally the last instance of this issue in the file is not duplicated with the correct name and just has the wrong name.

File: creature/badooga; Better Greatwyrms.json [hunk 3]
```diff
@@ -0,0 +1,2117 @@
...
+					"badooga"
+				],
+				"color": "c9c900",
+				"version": "1.0.0"
+			}
+		]
+	},
+	"monster": [
+		{
+			"name": "White Greatwyrm",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:22]: `group`s were recently migrated to arrays of strings.

File: creature/badooga; Better Greatwyrms.json [hunk 4]
```diff
@@ -0,0 +1,2117 @@
...
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "img/bestiary/MM/White%20Dragon.jpg"
+						}
+					}
+				]
+			},
+			"dragonCastingColor": "W",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:202]: Casting colours were recently migrated to be full strigns rather than abbreviations

**Review by revilowaldow — CHANGES_REQUESTED** [review]

File: creature/badooga; Better Greatwyrms.json [hunk 5]
```diff
@@ -0,0 +1,2117 @@
...
+				"color": "c9c900",
+				"version": "1.0.0"
+			}
+		]
+	},
+	"monster": [
+		{
+			"name": "White Greatwyrm",
+			"shortName": "greatwyrm",
+			"group": "Chromatic Dragon",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:24]:
```suggestion
			"group": [
			"Chromatic Dragon"
			],
```

File: creature/badooga; Better Greatwyrms.json [hunk 6]
```diff
@@ -0,0 +1,2117 @@
...
+							]
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Black Greatwyrm",
+			"shortName": "greatwyrm",
+			"group": "Chromatic Dragon",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:359]:
```suggestion
			"group": [
			"Chromatic Dragon"
			],
```

File: creature/badooga; Better Greatwyrms.json [hunk 7]
```diff
@@ -0,0 +1,2117 @@
...
+							]
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Green Greatwyrm",
+			"shortName": "greatwyrm",
+			"group": "Chromatic Dragon",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:689]:
```suggestion
			"group": [
			"Chromatic Dragon"
			],
```

File: creature/badooga; Better Greatwyrms.json [hunk 8]
```diff
@@ -0,0 +1,2117 @@
...
+							]
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Blue Greatwyrm",
+			"shortName": "greatwyrm",
+			"group": "Chromatic Dragon",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:1026]:
```suggestion
			"group": [
			"Chromatic Dragon"
			],
```

File: creature/badooga; Better Greatwyrms.json [hunk 9]
```diff
@@ -0,0 +1,2117 @@
...
+							]
+						}
+					]
+				}
+			]
+		},
+		{
+			"name": "Red Greatwyrm",
+			"shortName": "greatwyrm",
+			"group": "Chromatic Dragon",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:1370]:
```suggestion
			"group": [
			"Chromatic Dragon"
			],
```

File: creature/badooga; Better Greatwyrms.json [hunk 10]
```diff
@@ -0,0 +1,2117 @@
...
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "img/bestiary/MM/Blue%20Dragon.jpg"
+						}
+					}
+				]
+			},
+			"dragonCastingColor": "U",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:1205]:
```suggestion
			"dragonCastingColor": "blue",
```

File: creature/badooga; Better Greatwyrms.json [hunk 11]
```diff
@@ -0,0 +1,2117 @@
...
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "img/bestiary/FTD/Red%20Greatwyrm.webp"
+						}
+					}
+				]
+			},
+			"dragonCastingColor": "R",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:1535]:
```suggestion
			"dragonCastingColor": "red",
```

File: creature/badooga; Better Greatwyrms.json (hunk 4 above)
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:202]:
```suggestion
			"dragonCastingColor": "white",
```

File: creature/badooga; Better Greatwyrms.json [hunk 12]
```diff
@@ -0,0 +1,2117 @@
...
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "img/bestiary/MM/Black%20Dragon.jpg"
+						}
+					}
+				]
+			},
+			"dragonCastingColor": "B",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:523]:
```suggestion
			"dragonCastingColor": "black",
```

File: creature/badooga; Better Greatwyrms.json [hunk 13]
```diff
@@ -0,0 +1,2117 @@
...
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "img/bestiary/MM/Green%20Dragon.jpg"
+						}
+					}
+				]
+			},
+			"dragonCastingColor": "G",
```
Comment (revilowaldow) [inline creature/badooga; Better Greatwyrms.json:857]:
```suggestion
			"dragonCastingColor": "green",
```

**Review by revilowaldow — CHANGES_REQUESTED** [review]

**Comment by revilowaldow** [comment]
For future submissions, the team in #brew conversion can help you set up the schema to check your files before you submit.
The schema is what's checking your files when you pull them against the repo now anyway, and seeing it pop up with the full list of things to fix is handy.
image

**Review by TheGiddyLimit — APPROVED** [review]
powerful reviews

**Pull request merged**